bundles/binding/org.openhab.binding.knx/src/main/java/org/openhab/binding/knx/internal/dpt/KNXCoreTypeMapper.java

```diff
@@ -563,9 +549,9 @@ public String toDPTValue(Type type, String dptID) {
             // check for HSBType first, because it extends PercentType as well
             if (type instanceof HSBType) {
                 HSBType hc = ((HSBType) type);
-                return "r:" + Integer.toString(hc.getRed().intValue()) + " g:"
-                        + Integer.toString(hc.getGreen().intValue()) + " b:"
-                        + Integer.toString(hc.getBlue().intValue());
+                return "r:" + hc.getRed().intValue() + 
+                    + " g:" + hc.getGreen().intValue() + 
+                    + " b:" + hc.getBlue().intValue();
```

Comment:
This change is failing the build because you have contiguous '+' symbols:

 `+ + "g:"` and `+ + "b:"`

Reply:
Aside from the breaking change, the rest looks good to me.

Reply:
Thank You Chris.